storm-core/src/jvm/backtype/storm/topology/WindowedBoltExecutor.java

```diff
@@ -0,0 +1,200 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p/>
```

Comment:
minor nit. The `<p/>` is not in other license files, I think IntelleJ is adding it in for you.